extensions/applicationinsights-clickanalytics-js/README.md

```diff
@@ -0,0 +1,155 @@
+
+```html
+<script type="text/javascript" src="https://js.monitor.azure.com/scripts/b/ai.2.min.js"></script>
+<script type="text/javascript" src="cdnPathToCLickAnalyticsJS"></script>
+
+<script type="text/javascript">
+
+var clickPluginInstance = new Microsoft.ApplicationInsights.ClickAnalyticsPlugin();
+// Click Analytics configuration
+var clickPluginConfig = {
+    autoCapture : true
+}
+// Application Insights Configuration
+var configObj = {
+    instrumentationKey: "YOUR INSTRUMENTATION KEY",
+    extensions: [
+      clickPluginInstance
+    ],
+    extensionConfig: {
+        [clickPluginInstance.identifier] : clickPluginConfig
+    },
+};
+
+var appInsights = new Microsoft.ApplicationInsights.ApplicationInsights({ config: configObj });
+appInsights.loadAppInsights();
+
+</script>
+```
+
+## Configuration
+
+| Name | Type | Default | Description |
+|------|------|---------|-------------|
+| autoCapture | boolean | true | Automatic capture configuration. |
+| callback | IValueCallback | null | Callbacks configuration |
+| pageTags | string | null | Page tags |
+| dataTags | ICustomDataTags | null | Custom Data Tags provided to ovverride default tags used to capture click data. |
+| urlCollectHash | boolean | false | Enables the logging of values after a "#" character of the URL. |
+| urlCollectQuery | boolean | false | Enables the logging of the query string of the URL. |
+
+## IValueCallback
+
+| Name | Type | Default | Description |
+|------|------|---------|-------------|
+| pageName | Function | null | Function to override the default pageName capturing behavior. |
+| pageActionPageTags | Function | null | A callback function to augument the default pageTags collected during pageAction event. |
+| contentName | Function | null | A callback function to populate customized contentName. |
+
+## ICustomDataTags
+
+| Name | Type | Default | Description |
+|------|------|---------|-------------|
+| useDefaultContentName | boolean | false | When a particular element is not tagged with default customDataPrefix or customDataPrefix is not provided by user, this flag is used to collect standard HTML attribute for contentName. |
+| customDataPrefix | string | data- | Automatic capture content name and value of elements which are tagged with provided prefix. |
+| aiBlobAttributeTag | string | ai-blob | Click Analytics supports a JSON blob content meta data tagging instead of individual data-* attributes. |
+| metaDataPrefix | string | null | Automatic capture metadata name and content with provided prefix. |
+| captureAllMetaDataContent | string | null | Automatic capture all metadata names and content. Default is false. If enabled this will override provided metaTagPrefix. |
+| parentDataTag | string | null | Stop traversing up the DOM to capture content name and value of elements when encountered with this tag. |
+| donotTrackDataTag | string | ai-dnt | Custom attribute Tag to not track telemetry data. |
```

Comment:
Is it possible to rename this to "dntDataTag"? Or is this already used internally as this.